packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewerContainer.js

```diff
@@ -23,8 +23,7 @@ const DEFAULT_CELL_STYLE = {
   overlay: 'rgba(41,103,255,0.3)',
   selectedStroke: '#2967FF',
   selectedStrokeWidth: '8',
-  focusStroke: '#06FE76',
-  focusStrokeWidth: '4',
+  focusOutline: '2px dashed rgba(255, 255, 255, 0.5)',
```

Comment:
not blocking: the `focusOutline` (dashed lines) is easy to see on the smaller grids (larger images) and harder to see on the larger grids (smaller images). 

I'm wondering if we can modify the `focusOutline` between a clamping range of say 2..4(or6) based on the number total number of images in a grid via JS? 

Perhaps even just two types:
- large grid `focusOutlineLargeGrid: '4px dashed rgba(255, 255, 255, 0.5)',` (or 6px) 
- small grid `focusOutlineNormalGrid: '2px dashed rgba(255, 255, 255, 0.5)',`

Reply:
That might be possible with a calculated value in CSS too.

Reply:
There's some funky things you can do [with calc and css variables](https://www.kizu.ru/conditions-for-css-variables/) to get conditional outcomes... you can manage conditionals by way of media queries if one applies here too. In general, we're not leveraging newer CSS spec enough. [And maybe for the future if it actually gets proposed](https://tabatkins.github.io/specs/css-when-else/). Might not be worth it for readability and understandability though, so interpolating some JS into the CSS might be best.

Reply:
Lea Verou's been publishing a lot of good stuff over the past year or so. I think `min()`, `max()` and `clamp()` are in all the major browsers now. This is one of her tutorials with a hybrid JS/CSS approach to conditional styling: https://lea.verou.me/2020/06/hybrid-positioning-with-css-variables-and-max/